### ISSUE TITLE
xxhash.h not found

### ISSUE THREAD

**Issue by FormalSnake** [kovidgoyal/kitty#6539]
**Logs**
``
make                                                                                                                                                                                                                                                                                                                                                     
python3 setup.py
[8/34] Compiling kitty/charsets.c ... done
Compiling kittens/transfer/algorithm.c ...
clang -MMD -DNDEBUG -Wextra -Wfloat-conversion -Wno-missing-field-initializers -Wall -Wstrict-prototypes -std=c11 -pedantic-errors -Werror -O3 -fwrapv -fstack-protector-strong -pipe -fvisibility=hidden -D_FORTIFY_SOURCE=2 -flto -pthread -Ikitty -I/Library/Frameworks/Python.framework/Versions/3.10/include/python3.10 -I-I/opt/homebrew/Cellar/xxhash/0.8.2/include -c kittens/transfer/algorithm.c -o build/rsync-algorithm.c.o
kittens/transfer/algorithm.c:13:10: fatal error: 'xxhash.h' file not found
#include <xxhash.h>
         ^~~~~~~~~~
1 error generated.
make: *** [all] Error 1
``
**Environment details**
```
MacOS Sonoma public beta 2
```

**Comment by kovidgoyal** [comment]
Install xxhash

**Comment by FormalSnake** [comment]
I did brew install xxhash, and it doesn't work

brew install xxhash                                       
Running `brew update --auto-update`...
==> Auto-updated Homebrew!
Updated 2 taps (homebrew/core and homebrew/cask).
==> New Formulae
katana                          uncover
==> New Casks
git-credential-manager

You have 45 outdated formulae and 5 outdated casks installed.

Warning: xxhash 0.8.2 is already installed and up-to-date.
To reinstall 0.8.2, run:
  brew reinstall xxhash
